ecc_enclave/enclave/enclave.edl

```diff
@@ -29,6 +29,14 @@ enclave {
                 [out, size=max_dn_len] char *dn, uint32_t max_dn_len,
                 [user_check] void *u_shim_ctx);
 
+        void ocall_get_channel_id(
+                [out, size=max_channel_id_len] char *channel_id, uint32_t max_channel_id_len,
+                [user_check] void *u_shim_ctx);
+
+        void ocall_get_msp_id(
+                [out, size=max_msp_id_len] char *msp_id, uint32_t max_msp_id_len,
+                [user_check] void *u_shim_ctx);
```

Comment:
an alternative to implement each of these shim methods as ocall, we could also implement `ocall_fetch_cc_params` which is called only if the cc_params are not already loaded and stored in the enclave as they are "static" values anyway.
Implementing `fetch_cc_params` would also better align with the current design described in the UMLs https://github.com/hyperledger-labs/fabric-private-chaincode/blob/concept-release-2.0/docs/design/fabric-v2%2B/fpc-registration.puml#L65

Reply:
However, my personal preference is to include this data as parameter in the `createEnclave` call and keep inside the enclave as non-mutual data.